security/pkg/nodeagent/test/csr_failure/csr_test.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 func TestCSRFailure(t *testing.T) {
+	t.Skip("https://github.com/istio/istio/issues/24220")
```

Comment:
The title of the PR says "enable test ..." but it seems it's actually skipping the tests? (TestCSRFailure and TestBadCSRResponse), can you update the title to match the code change?

Reply:
update it